Blog post about our status page (#496)
2i2c-org/2i2c-brochure: content/blog/2025/status-page/index.md

---
title: "Demonstrating our infrastructure's reliability with a hub status page for our communities"
subtitle: ""
tags: ["reliability"]
categories: ["service"]
date: "2025-09-23"
featured: false
draft: false
---

One of 2i2c's goals is to **make the cloud safe for science**.

> A big part of this is making the black box of commercial cloud infrastructure more predictable and reliable, while running a network of community hubs that all operate autonomously.

Reviewer: ```suggestion
A big part of this is making the black box of commercial cloud infrastructure more predictable and reliable for our member communities, across our network of community hubs that all operate autonomously.
```

> As part of this effort, we've created a **status page for 2i2c's network of community hubs**. This is a source of truth to provide a high-level picture of the stability of our infrastructure, and to give us a heads up when things aren't working as expected. You can check it out at:

Reviewer: ```suggestion
To that end, we've created a **status page for 2i2c's network of community hubs**. This is a source of truth to provide a high-level picture of the stability of our infrastructure, let a community know if their hub is experiencing a problem, and to give us a heads up when things aren't working as expected. You can check it out at:
```

👉 [**`status.2i2c.org`**](https:status.2i2c.org)

{{< figure src="featured.png" caption="The 2i2c Status Page gives communities a high-level view of the uptime for our entire network of community hubs.">}}

> As part of this process, we've also [streamlined our incident response processes](https://github.com/2i2c-org/team-compass/pull/1021) in order to more quickly respond to outages when they occur (ideally, before a community has even noticed!).

Reviewer: ```suggestion
While we make status more visible, we're also [streamlining our incident response processes](https://github.com/2i2c-org/team-compass/pull/1021) in order to more quickly respond to outages when they occur (ideally, before a community has even noticed!).
```

> There are still plenty of improvements we'd like to make: for example, we're focusing on major outages right now, but would like to extend some level of reporting for _degraded_ service, like unexpectedly slow start times.

Reviewer: I'd love feedback from communities on this one - should we use the CSAT form? Have them contact us via support@2i2c.org?

```suggestion
There are still plenty of improvements we'd like to make: for example, we're focusing on major outages right now, but would like to extend some level of reporting for _degraded_ service, like unexpectedly slow start times. We'd love to hear feedback. If you use our network of hubs and have suggestions for status page improvements, reach out to our [support team](https://docs.2i2c.org/support/).
```

Author: I added a button for feedback at the top, so I'll not merge this change in!
